pkg/operator/k8sutil/k8sutil.go

```diff
@@ -0,0 +1,68 @@
+/*
+Copyright 2016 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+Some of the code below came from https://github.com/ceph/go-ceph
+which has the following license:
+
+Copyright 2016 The etcd-operator Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
if the etcd-operator code uses the exact same license (apache 2.0), we may not need to repeat the license again a 2nd time here (just the attribution note is perhaps all that's needed)

this comment would apply to other places we are using etcd-operator code.